staticjinja/staticjinja.py

```diff
@@ -67,24 +72,28 @@ def resolve_path(path):
        See https://github.com/staticjinja/staticjinja/issues/149
     """
     if Path(path).is_absolute():
-        return path
+        return str(path)
 
     project_root = Path.cwd()
     build_script_directory = get_build_script_directory()
-    if build_script_directory is not None:
-        if build_script_directory != project_root:
-            warnings.warn(
-                FutureWarning(
-                    "Inferring project root to be the build script directory is "
-                    "deprecated. For more info see "
-                    "https://github.com/staticjinja/staticjinja/issues/149"
-                ),
-                stacklevel=2,  # Show the caller of resolve_path() in error
-            )
+    if build_script_directory != project_root:
```

Comment:
Nice catch on how to simplify

Reply:
I actually did this because I made `get_build_script_directory()` return `Path.cwd()` if running from a REPL. It makes it easier when updating the tests.

Reply:
Oh I didn't see that before, I like this. Will rename `get_build_script_directory` and fix the docstring in a fixup